gmso/external/convert_mbuild.py

```diff
@@ -85,7 +85,7 @@ def from_mbuild(compound, box=None, search_method=element_by_symbol):
             for particle in child.particles():
                 pos = particle.xyz[0] * u.nanometer
                 ele = search_method(particle.name)
-                site = Site(name=particle.name, position=pos, element=ele)
+                site = Atom(name=particle.name, position=pos, element=ele)
```

Comment:
This can be another PR, but we can do the checking when converting from mbuild and see if we want to create a GMSO bead or atom (now that Compound has that element attribute)

Reply:
Got it. This might be better to use after that version of mbuild is released